packages/perseus/src/widgets/matrix.tsx

```diff
@@ -503,16 +503,14 @@ class Matrix extends React.Component<Props, State> {
         value,
         cb,
     ) => {
-        const answers = _.map(this.props.answers, _.clone);
+        const answers = this.props.answers.map((answer) => [...answer]);
```

Comment:
Switching from `_` to plain ole' JS here enables TS to understand the type of our local `answers` variable... which allows more removal of `@ts-expect-error` here.